packages/appstore/src/components/main-title-bar/asset-summary.tsx

```diff
@@ -24,6 +29,8 @@ const AssetSummary = observer(() => {
     const platform_real_balance = useTotalAccountBalance(platform_real_accounts);
     const cfd_real_balance = useTotalAccountBalance(cfd_real_accounts);
     const cfd_demo_balance = useTotalAccountBalance(cfd_demo_accounts);
+    const total_assets_real_currency = useTotalAssetCurrency() ?? 'USD';
```

Comment:
what about we don't set a fallback/default currency!
It has it's own drawbacks tho as we've seen recently if there's no currency set

Reply:
@shahzaib-deriv My big fault! Nice catch, I've added this to satisfy TS because we did this on one of the hooks. Fixed the issue!